src/main/java/org/myrobotlab/service/abstracts/AbstractMotor.java

```diff
@@ -162,59 +147,39 @@ public boolean isAttached(MotorController controller) {
 
   @Override
   public boolean isInverted() {
```

Comment:
this should be moved to the Mapper class , right?

Reply:
I don't believe so .. I moved the fields and decomposed Mapper into AbstractMotor - 
Mapper I think is flawed in a couple ways, setting or re-setting map values will remove min/max.
I think Mapper does more than it should and the cost is removal of min/max when creating new map values.  I didn't want to risk cause problems to Servos (which uses mappers), which I don't want to touch now.

I also don't want to bury data in sub-components ... 

In general terms,
     The MotorController will use the MotorControl data to correctly drive a motor...

Reply:
So, if i want to swap in a new non-linear mapper, how would I do that ?  
ultimately, I want to just say MotorContol.setMapper(myMapper)  and it should work.
the interface really only needs to know about calcOutput ...